python/main.py

```diff
@@ -18,14 +26,87 @@
     allow_headers=["*"],
 )
 
+#For saving into JSON file
+'''
+def write_json(new_item, filename='items.json'):
+    open(filename, 'a').close()
+    if os.stat(filename).st_size == 0:
+        with open(filename, 'w') as fp:
+            res = {"items":[new_item]}
+            json.dump(res,fp)
+    else:
+        with open(filename,'r+') as fp:
+            file_data = json.load(fp)
```

Comment:
Nitpick: why don't you name the variable `res` ? that way it's the same with the condition above this

Reply:
> Nitpick: why don't you name the variable `res` ? that way it's the same with the condition above this

That's true. I will change the name of the variable to res.